nest-forms-backend/src/auth-v2/utils/user-utils.ts

```diff
@@ -0,0 +1,51 @@
+import { FormOwnerType } from '@prisma/client'
+import {
+  UserVerifyStateCognitoTierEnum,
+  UserVerifyStateTypeEnum,
+} from 'openapi-clients/city-account'
+
+import { isAuthUser, User } from '../types/user'
+
+const tierVerifiedMap = {
```

Comment:
This might seem weird, but it assures all `UserVerifyStateCognitoTierEnum` are covered, when someone adds new `UserVerifyStateCognitoTierEnum` it won't compile anymore.